package.json

```diff
@@ -11,15 +11,39 @@
     "package:js": "lerna run package:js && tools/collect-dist.sh",
     "package-windows": "lerna run package && tools\\collect-dist.bat",
     "bootstrap-plugin-cache": "./test/run-against-dist ./tools/bootstrap-plugin-cache.sh",
-    "examples:reinstall": "lerna run --parallel --scope @examples/* --ignore @examples/java* reinstall && lerna run --stream --scope @examples/java* reinstall --concurrency 1",
-    "examples:build": "lerna run --parallel --scope @examples/* --ignore @examples/java* build && lerna run --stream --scope @examples/java* build --concurrency 1",
-    "examples:synth": "lerna run --parallel --scope @examples/* --ignore @examples/java* synth && lerna run --stream --scope @examples/java* synth --concurrency 1",
+    "examples:reinstall": "lerna run --parallel --scope @examples/* reinstall",
+    "examples:reinstall:java": "lerna run --parallel --scope @examples/java* reinstall",
```

Comment:
Running the Java tests in parallel may result in occasional failures since I don't believe the local maven repository is thread safe. Pretty sure I ran into issues when initially adding Java.

Reply:
ok, that would be bad. I think Java takes the most time. But yeah, let's see how long this takes sequentially.

Reply:
```
[  233.000s ]  ├──────────────────┤                                                            * build (0.13.4)-2005821848
[  880.000s ]  ├───────────────────────────────────────────────────────────────────────────┤   * build (0.14.0, java)-2005821833
[  205.000s ]  ├───────────────┤                                                               * build (0.12.29)-2005821848
[  472.000s ]  ├───────────────────────────────────────┤                                       * build (0.14.0, typescript)-2005821833
[  659.000s ]  ├───────────────────────────────────────────────────────┤                       * build (0.14.0, csharp)-2005821833
[  226.000s ]  ├─────────────────┤                                                             * build (0.14.0)-2005821848
[  232.000s ]                       ├──────────────────┤                                       * linux_integration (0.12.29, java)-2005821848
[  346.000s ]                       ├────────────────────────────┤                             * linux_integration (0.12.29, typescript)-2005821848
[  229.000s ]                       ├──────────────────┤                                       * linux_integration (0.13.4, python)-2005821848
[  223.000s ]                       ├─────────────────┤                                        * linux_integration (0.12.29, python)-2005821848
[  337.000s ]                       ├───────────────────────────┤                              * linux_integration (0.14.0, typescript)-2005821848
[  224.000s ]                       ├─────────────────┤                                        * linux_integration (0.13.4, csharp)-2005821848
[  226.000s ]                       ├─────────────────┤                                        * linux_integration (0.14.0, python)-2005821848
[  348.000s ]                       ├────────────────────────────┤                             * linux_integration (0.13.4, typescript)-2005821848
[  222.000s ]                       ├─────────────────┤                                        * linux_integration (0.14.0, csharp)-2005821848
[  338.000s ]                       ├───────────────────────────┤                              * windows_integration (0.12.29, csharp)-2005821848
[  431.000s ]                       ├───────────────────────────────────┤                      * windows_integration (0.12.29, java)-2005821848
[  208.000s ]                       ├────────────────┤                                         * windows_integration (0.12.29, typescript)-2005821848
[  233.000s ]                       ├──────────────────┤                                       * linux_integration (0.12.29, csharp)-2005821848
[  274.000s ]                       ├──────────────────────┤                                   * linux_integration (0.13.4, java)-2005821848
[  270.000s ]                       ├─────────────────────┤                                    * linux_integration (0.14.0, java)-2005821848
[  264.000s ]                       ├─────────────────────┤                                    * windows_integration (0.12.29, python)-2005821848
[  331.000s ]                       ├───────────────────────────┤                              * windows_integration (0.13.4, csharp)-2005821848
[  288.000s ]                       ├───────────────────────┤                                  * windows_integration (0.14.0, python)-2005821848
[  193.000s ]                       ├──────────────┤                                           * windows_integration (0.13.4, typescript)-2005821848
[  377.000s ]                       ├───────────────────────────────┤                          * windows_integration (0.14.0, csharp)-2005821848
[  204.000s ]                       ├───────────────┤                                          * windows_integration (0.14.0, typescript)-2005821848
[  396.000s ]                       ├────────────────────────────────┤                         * windows_integration (0.14.0, java)-2005821848
[  291.000s ]                       ├───────────────────────┤                                  * windows_integration (0.13.4, python)-2005821848
[  374.000s ]                        ├──────────────────────────────┤                          * windows_integration (0.13.4, java)-2005821848

     one '.' unit is less than:   11.404s
                    total time:  912.320s
```

where java examples are running sequentially now. 880s (sequentially) vs 677s (parallel). I think I'd try the parallel version and see if there are issues popping up. What do you think @jsteinich?

Reply:
Sounds good.

Reply:
A sidenote: All of these tests are running at least 50% faster on my local machine, so that's something to keep in mind when improving this further.